src/sql/workbench/api/common/sqlExtHostTypes.ts

```diff
@@ -320,6 +321,17 @@ export enum AzureResource {
 	Sql = 1
 }
 
+export interface ServerInfoOption {
+	isBigDataCluster: boolean;
```

Comment:
I thought we discussed not special casing these in core? The core of ADS has no reason to know this, it's all special cased and only needed in the mssql extension. The fact we're relying on these is not a good sign.